front-end/src/Components/Config/config.tsx

```diff
@@ -0,0 +1 @@
+export const config = 'http://10.50.80.56:5100/api';
```

Comment:
Could we have this as an environmental variable? https://newbedev.com/using-process-env-in-typescript